lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -739,8 +747,8 @@ def add_placeholder_target
           #
           def add_swift_static_library_compatibility_header_phase(native_target)
             if custom_module_map
-              raise Informative, 'Using Swift static libraries with custom module maps does not yet work.' \
-                                 "Please build #{pod_target.label} as a framework or remove the custom module map for the time being."
+              raise Informative, 'Using Swift static libraries with custom module maps does not yet work. ' \
+                                 "Please build `#{target.label}` as a framework or remove the custom module map for the time being."
```

Comment:
@segiddins this was actually crashing. `pod_target` is not a declared variable here.

Reply:
thankfully a different test caught it.